bukkit/src/main/java/co/aikar/commands/BukkitCommandContexts.java

```diff
@@ -241,7 +241,7 @@ public BukkitCommandContexts(BukkitCommandManager manager) {
     }
 
     @Nullable
-    OnlinePlayer getOnlinePlayer(BukkitCommandIssuer issuer, String lookup, boolean allowMissing) throws InvalidCommandArgument {
+    protected OnlinePlayer getOnlinePlayer(BukkitCommandIssuer issuer, String lookup, boolean allowMissing) throws InvalidCommandArgument {
```

Comment:
I don't want to call this method,  I want to override it.

Reply:
I would rather you replace the default Context Resolvers (which when you replace a context resolver, it will return the original, so you will have the original resolver if you need), and reimplement them with your desired logic.